### ISSUE TITLE
Upgrade to React 19

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#3452]
#3445

**Comment by corneliusroemer** [comment]
New PR: https://github.com/loculus-project/loculus/pull/3597

**Comment by anna-parker** [comment]
blocked by the MUI material dependency: https://github.com/loculus-project/loculus/issues/1548
